content/en/blog/_posts/2024-04-29-structured-authentication-configuration-beta.md

```diff
@@ -0,0 +1,219 @@
+---
+layout: blog
+title: "Kubernetes 1.30: Structured Authentication Configuration Moves to Beta"
+date: 2024-04-29
```

Comment:
Per Tim's comment below and Slack discussion, we want to update the publishing schedule so this one publishes a day before the authz blog post.
```suggestion
date: 2024-04-25
```

Reply:
thanks @kcmartin, i've updated the date.